web/src/composables/useLogs.ts

```
@@ -4342,6 +4410,726 @@ const useLogs = () => {
+      ) {
+        queryReq.query["query_fn"] =
+          b64EncodeUnicode(searchObj.data.tempFunctionContent) || "";
+      }
+
+      if (searchObj.data.datetime.type === "relative") {
+        if (!isPagination) initialQueryPayload.value = cloneDeep(queryReq);
+      } else {
+        if (
+          searchObj.meta.refreshInterval == 0 &&
+          router.currentRoute.value.name == "logs" &&
+          searchObj.data.queryResults.hasOwnProperty("hits")
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)